bundles/org.openhab.binding.yamahamusiccast/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="yamahamusiccast"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
+	<thing-type id="Device">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+		<label>Yamaha MusicCast Model</label>
+		<description>Your Yamaha model with MusicCast functionality</description>
+
+		<channel-groups>
+			<channel-group id="main" typeId="mainControls"/>
+			<channel-group id="zone2" typeId="zone2Controls"/>
+			<channel-group id="zone3" typeId="zone3Controls"/>
+			<channel-group id="zone4" typeId="zone4Controls"/>
+			<channel-group id="playerControls" typeId="playerControls"/>
+		</channel-groups>
+
+		<config-description>
+			<parameter name="configHost" type="text" required="true">
+				<label>Address</label>
```

Comment:
You could specify `<context>network-address</context>` to get a free validation.

Reply:
Done, available via push